docs/howtos/audit-scanner.md

```diff
@@ -0,0 +1,52 @@
+---
+sidebar_label: "Audit Scanner Installation"
+title: "Kubewarden Audit Scanner Installation"
+---
+
+# Kubewarden Audit Scanner installation
+
+Starting from version `v1.7.0`, Kubewarden introduces a new component called
+the Audit Scanner. This scanner evaluates whether the resources running in the
+cluster comply with the deployed policies. It is particularly useful when
+policies are deployed after the resources. The Audit Scanner allows operators
+to detect resources deployed before the policies that violate the validations
+performed by the policies.
+
+## Installation
+
+To install and use the Kubewarden Audit Scanner, please follow these steps:
+
+1. Install the `kubewarden-controller` Helm chart, ensuring you have version
+   `v1.7.0` or higher. Remember to enable the audit scanner component.
+
+   ```console
+   helm install --set experimental.auditScanner.enable=true kubewarden-controller kubewarden/kubewarden-controller
+   ```
+
+   For more information about the installation of Kubewarden see the [Quick Start guide](../quick-start.md)
+
+2. Install the Policy Report CRDs, which are required for the Audit Scanner to
```

Comment:
this isn't needed, they are included by default on `kubewarden-crds`, unless one sets  `installPolicyReportCRDs` to `false`.